### DISCUSSION TITLE
Chart config improvements: what would you like to see?

### DISCUSSION THREAD

**Discussion by TuringLovesDeathMetal** [lightdash/lightdash#1176] [Ideas]
Charts are a really important part of Lightdash, and we know they could use a little love ❤️‍🔥 

So, we'd love to hear it from you:
> What chart configurations are missing from Lightdash?

([here's the list of issues we have logged right now](https://github.com/lightdash/lightdash/milestone/23))

--> Have general feedback about the chart configurations? pop them in a comment! 
--> want to suggest a chart type that we're missing? [Check out this discussion instead.](https://github.com/lightdash/lightdash/discussions/1175)
--> Have a few minutes to answer some questions? Check out our questions below and stick your answers in a comment.

We really appreciate your help! 🤗 

### ❓ Some questions we have for you:
1. How often do you use the default chart configurations in your BI tool? (or in Lightdash)
2. In the last chart you built, what configurations did you apply? 
3. Was there anything missing/configurations that you couldn't do that you wish you could have done?

**Comment by djayatillake** [comment]
Generally it would be good to configure each series on the chart independently: diff chart type, additional y axes, manual colour choice, possibly different legends per series.

**Reply by TuringLovesDeathMetal** [reply]
all great suggestions!

**Comment by risafangs** [comment]
I'd like to be able to put the value of a data point above it on the visualization - like on a bar chart, show count or other metric on top of the bar for that x axis value.
